packages/react-composites/tests/browser/call/CallComposite.test.ts-snapshots/composite pages load completely

```diff
@@ -0,0 +1 @@
+{"getPage-parent rerender":1,"-parent rerender":126,"selectLocalPreview-parent rerender":29,"getLocalMicrophoneEnabled-parent rerender":29,"-return state changed":24,"selectLocalPreview-return state changed":4}
```

Comment:
This is the re-render snapshot for each selector-event - some selectors are anonymous so we can't track the name, so we need to make changes like in localPreviewSelector to give them a name